tests/src/Kernel/Entity/FieldOverrideTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal\Tests\og\Kernel\Entity\FieldOverrideTest.
+ */
+
+namespace Drupal\Tests\og\Kernel\Entity;
+
+use Drupal\Component\Utility\Unicode;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\node\Entity\NodeType;
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\og\Og;
+
+/**
+ * Testing field definition overrides.
+ *
+ * @group og
+ */
+class FieldOverrideTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = ['user', 'field', 'entity_reference', 'node', 'og', 'system'];
```

Comment:
Can we test is on a non-node entity - just for the sake of making sure we don't have node centric stuff going on.